### PR TITLE
Enforce snake_case in token

### PR DESCRIPTION
**Pull request by glenacota**
Changes in this PR attempt to solve issue #4746, using the snake case in tokens where it is not.

~- [ ] The page (if new), does not already exist in the repo.~
~- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)~
~- [ ] The page has 8 or fewer examples.~
~- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).~
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
~- [ ] The page description includes a link to documentation or a homepage (if applicable).~

### CODE REVIEW

File: pages/common/nohup.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 - Run process that can live beyond the terminal:
 
-`nohup {{command options}}`
+`nohup {{command_with_options}}`
```
Comment (navarroaxel) [inline pages/common/nohup.md:7]: Yeah, is definitively better. What about this is option?
```suggestion
`nohup {{command}} {{command_options}}`
```

File: pages/windows/azcopy.md [hunk 2]
```diff
@@ -9,19 +9,19 @@
 
 - Upload a local file:
 
-`azcopy copy '{{path/to/source/file}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}/{{blob-name}}'`
+`azcopy copy '{{path/to/source/file}}' 'https://{{storage_account_name}}.blob.core.windows.net/{{container_name}}/{{blob_name}}'`
```
Comment (navarroaxel) [inline pages/windows/azcopy.md:12]: Maybe this was made because `_` is not valid in URLs. But we don't need that the example URL works :man_shrugging:

**Review by navarroaxel — APPROVED** [review]

File: pages/common/nohup.md (hunk 1 above)
Comment (sbrl) [reply pages/common/nohup.md:7]: Either would work.

File: pages/windows/azcopy.md (hunk 2 above)
Comment (sbrl) [reply pages/windows/azcopy.md:12]: It isn't? I didn't know that. In that case, we should revert this one.

**Review by sbrl — COMMENTED** [review]
Looks ok to   me after @navarroaxel's comments are addressed

File: pages/common/nohup.md (hunk 1 above)
Comment (glenacota) [reply pages/common/nohup.md:7]: I found in the codebase three `{{command}} {{command_arguments}}` , one `{{command}} {{args}}`, and a couple `{{args}}`. Just to add some more doubts :)

File: pages/windows/azcopy.md (hunk 2 above)
Comment (glenacota) [reply pages/windows/azcopy.md:12]: Underscores in URLs are valid, e.g., https://en.wikipedia.org/wiki/Main_Page. I don't see any special reason not to use them :)

Comment (navarroaxel) [reply pages/windows/azcopy.md:12]: I mean for the subdomain, you can't use underscore (`_`) but you can use hyphens (`-`)

Comment (glenacota) [reply pages/windows/azcopy.md:12]: You're right, but then only the first token would require the hyphen. Of course, it'd be very ugly to use hyphens and underscores in the same example and we should stick to one :) IMHO: validity shouldn't be an issue with tokens

Comment (sbrl) [reply pages/windows/azcopy.md:12]: Generally we prefer underscores in token syntax elsewhere, so for consistency I think underscores might be the better choice here.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @glenacota!

**Pull request merged**